static/js/invite.js

```diff
@@ -208,6 +215,7 @@ exports.initialize = function () {
     $("#invite-user").on("change", "#generate_multiuse_invite_radio", () => {
         $("#invitee_emails").prop("disabled", false);
         $("#submit-invitation").text(i18n.t("Invite"));
+        $("#copy_generated_invite_link").attr("type", "hidden");
```

Comment:
Shouldn't this be `.hide()`  And for the other ones `.show()`?

Reply:
Yeah we can do that, sorry missed out on that one.😅

Reply:
@timabbott but for the .show() we are changing the input type to button, .show() won't be applicable there?